### PR TITLE
[cypher/uk-ua] Add ukrainian translation for Cypher query language

### PR DESCRIPTION
**Pull request by AstiaSun**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: uk-ua/cypher-ua.html.markdown [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+`(p:Person)`
+Можна також додати *ярлик* до вершини, в данному випадку - **Person**. Це як тип / клас / категорія. Назва *ярлика* починається з великої літери та викоритовує верблюжу нотацію.
+
+`(p:Person:Manager)`
+Вершина може мати кілька *ярликів*.
+
+`(p:Person {name : 'Théo Gauchoux', age : 22})`
+Вершина також може мати різні *властивості*, в данному випадку - **name** та **age**. Також мають починаєтися з великої літери та використовувати верблюжу нотацію.
+
+Наступні типи дозволяється викоритовувати у властивостях:
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: використовувати

File: uk-ua/cypher-ua.html.markdown [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+Зв'язки (або ребра)
+---
+
+**Сполучають дві вершини**
+
+`[:KNOWS]`
+Це *зв'язок* з *ярликом* **KNOWS**. Це такий же самий *ярлик* як і у вершини. Починається з великої літери та використовує ВЕРХНІЙ\_РЕГІСТР\_ІЗ\_ЗМІЇНОЮ\_НОТАЦІЄЮ.
+
+`[k:KNOWS]`
+Те же самий *зв'язок*, до якого можна звертатись через змінну **k**. Можна подалі використовувати у запиті, хоч це і не обов'язково.
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: Той же?

File: uk-ua/cypher-ua.html.markdown [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+**Сполучають дві вершини**
+
+`[:KNOWS]`
+Це *зв'язок* з *ярликом* **KNOWS**. Це такий же самий *ярлик* як і у вершини. Починається з великої літери та використовує ВЕРХНІЙ\_РЕГІСТР\_ІЗ\_ЗМІЇНОЮ\_НОТАЦІЄЮ.
+
+`[k:KNOWS]`
+Те же самий *зв'язок*, до якого можна звертатись через змінну **k**. Можна подалі використовувати у запиті, хоч це і не обов'язково. 
+
+`[k:KNOWS {since:2017}]`
+Те й же *зв'язок*, але вже із *властивостями* (як у *вершини*), в данному випадку властивіть - це  **since**.
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: Той же? (або теж?); властивість

File: uk-ua/cypher-ua.html.markdown [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+`[k:KNOWS*..4]`
+Це стуктурна інформація, яку використовують *шляхи*, які розглянуті нижче. В данному випадку, **\*..4** говорить: "Сумістити шаблон із зв'язком **k**, що повторюватиметься від одного до чотирьох разів."
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: структурна

File: uk-ua/cypher-ua.html.markdown [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+`(a:Person)-[:MANAGES]->(b:Person)`
+Шлях може бути направленим. Цей описує, що **а** є менеджером **b**.
+
+`(a:Person)-[:KNOWS]-(b:Person)-[:KNOWS]-(c:Person)`
+Можна створювати ланцюги зі зв'язків. Цей шлях описує друга друга (**a** знає **b**, який в свою чергу знає **c**).
+
+`(a:Person)-[:MANAGES]->(b:Person)-[:MANAGES]->(c:Person)`
+Ланцюг, аналогічно, також може бути направленим. Шлях описує, що **a** -  бос **b** і супер бос для **c**.
+
+Шаблони, які часто викоритовуються (з документації Neo4j):
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: використовуються

File: uk-ua/cypher-ua.html.markdown [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+RETURN n
+```
+
+Знайти вершини за ярликом:
+```
+MATCH (a:Person)
+RETURN a
+```
+
+Знайти вершини за ярликом та властивостю:
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: властивістю? (Навіть у словнику подивився)))

File: uk-ua/cypher-ua.html.markdown [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+```
+MATCH (a:Person {name:"Théo Gauchoux"})
+RETURN a
+```
+
+Знайти вершини відовідно до зв'язків (ненаправлених):
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: відповідно

File: uk-ua/cypher-ua.html.markdown [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+```
+MATCH (a)-[:KNOWS]-(b)
+RETURN a,b
+```
+
+Знайти вершини відовідно до зв'язків (направлених):
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: відповідно

File: uk-ua/cypher-ua.html.markdown [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+```
+MATCH (a)-[:MANAGES]->(b)
+RETURN a,b
+```
+
+Знайти вершини зо допомогою `WHERE`:
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: за допомогою

File: uk-ua/cypher-ua.html.markdown [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+```
+
+Оновити всі властивості вершини:
+```
+MATCH (p:Person)
+WHERE p.name = "Théo Gauchoux"
+SET p = {name: "Michel", age: 23}
+```
+
+Додати нову властивіть до вершини:
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: властивість

File: uk-ua/cypher-ua.html.markdown [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+MATCH (p:Person)
+WHERE p.name = "Théo Gauchoux"
+SET p:Internship
+```
+
+
+Запити на видалення
+---
+
+Зидалити окрему вершину (пов'язані ребра повинні бути видалені перед цим):
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: Видалити

File: uk-ua/cypher-ua.html.markdown [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+`LIMIT x`
+Обмежити результат до x перших результатів.
+
+
+Особливі підказки
+---
+
+- У мові Cypher існують лише однорядкові коментарі, що позначаються двійним слешем : // Коментар
+- Можна виконати скрипт Cypher, збережений у файлі **.cql** прямо в Neo4j (прямо як імпорт). Проте, не можна мати мати кілька виразів в цьому файлі (розділених **;**).
+- Використовуйте команднйи рядок Neo4j для написання запитів Cypher, це легко і швидко.
```
Comment (Menelion) [inline uk-ua/cypher-ua.html.markdown]: командний

**Comment by AstiaSun** [comment]
@Menelion Thanks so much for your review! Changes have been committed.

**Comment by Menelion** [comment]
@AstiaSun Thanks so much! 👍

**Pull request merged**